src/Overlay/Overlay.js

```diff
@@ -431,6 +431,13 @@ module.exports.Selection = {
 	/**
 	* @private
 	*/
+	handlers: {
+		onSpotlightFocused: 'spotlightFocused'
```

Comment:
It's really not safe to use the handlers block from a mixin because you'll likely stomp on the target instances handlers. That'll happen here with GridListImageItem which already has a onSpotlightFocused handler. You'll probably need to do something like MarqueeSupport does to be safe.